packages/teleport/src/DesktopSession/useTdpClientCanvas.tsx

```diff
@@ -83,6 +84,7 @@ export default function useTdpClientCanvas(props: Props) {
   const onClipboardData = (clipboardData: ClipboardData) => {
     if (clipboardSharingEnabled && document.hasFocus() && clipboardData.data) {
       navigator.clipboard.writeText(clipboardData.data);
+      latestClipboardText.current = clipboardData.data;
```

Comment:
Would computing a checksum make more sense here? I'm thinking about copying large amounts of data - this requires the browser to hold an entire copy of the data in memory just to tell if the data changes in the future.

Reply:
Right, ok I added a Sha256 digest 46abb46df277209401fc36f69fbea3f11b4ed8fd